G/GslibIO/Compat.toml

```diff
@@ -7,6 +7,10 @@ julia = "0.5-0"
 ["0.1"]
 julia = "0.6-0"
 
-["0.2-0"]
+["0.2.0"]
 FileIO = "1"
 julia = "0.7-1"
+
+["0.2.1-0"]
+FileIO = "1.0.0-*"
```

Comment:
Why not just `1`? Same for julia

Reply:
I am always confused with these version numbers. Could you please merge for now? I will take care of all these issues in the near future.

Reply:
https://github.com/juliohm/GslibIO.jl/pull/2